### ISSUE TITLE
PI <?dbsuse-fo font-size="7pt"?> adds an additional line break in screen

### ISSUE THREAD

**Issue by sknorr** [openSUSE/suse-xsl#53]
In screens, the processing instruction `<?dbsuse-fo font-size="7pt"?>` creates an additional line break at the beginning. That is ugly/undesirable.

Example:
```
<screen><?dbsuse-fo font-size="7pt"?>&prompt.user;vmstat 2
 procs -----------memory---------- ---swap-- -----io---- -system-- ------cpu-----
 r  b   swpd   free   buff  cache   si   so    bi    bo   in   cs us sy id wa st</screen>
```

PDF Output:
```

tux > vmstat 2
 procs -----------memory---------- ---swap-- -----io---- -system-- ------cpu-----
 r  b   swpd   free   buff  cache   si   so    bi    bo   in   cs us sy id wa st
```

(The PI itself is quite the hack, but, well, it is there and may need to be supported...)

**Comment by tomschr** [comment]
Test with Saxon and upstream stylesheets. Good luck! :+1:

**Comment by sknorr** [comment]
This should be fixed with eba1e8b18f40b5201b900a7c712681e410a1eaf6, c914e4305aed9159bccdd34c2ce90d983626c49a, and 53708567d217b3037be6e690955898b666a19476.
